papaya-core/src/main/java/pl/edu/agh/papaya/service/jira/JiraOAuthTokenFactory.java

```diff
@@ -0,0 +1,56 @@
+package pl.edu.agh.papaya.service.jira;
+
+import com.google.api.client.auth.oauth.OAuthRsaSigner;
+import com.google.api.client.http.apache.ApacheHttpTransport;
+import com.google.api.client.util.Base64;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+
+class JiraOAuthTokenFactory {
+
+    private final transient String accessTokenUrl;
+    private final transient String requestTokenUrl;
+
+    JiraOAuthTokenFactory(String jiraBaseUrl) {
+        this.accessTokenUrl = jiraBaseUrl + "/plugins/servlet/oauth/access-token";
+        requestTokenUrl = jiraBaseUrl + "/plugins/servlet/oauth/request-token";
```

Comment:
Celowo jeden ma `this`, a drugi nie?

Reply:
Dodano brakujące `this` wspierając przyjętą konwencje